### PR TITLE
Update the README

### PR DESCRIPTION
**Pull request by ejizba**
There were several places that were out of date or not entirely accurate.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,46 +2,33 @@
...
+- Run "Extensions: Show Recommended Extensions" from the [command palette](https://code.visualstudio.com/docs/getstarted/userinterface#_command-palette) and install all extensions listed under "Workspace Recommendations"
+- Run `npm install` and `npm run build`
+- Create or open a local function app to test with
+- In the local function app, add the following settings to your "local.settings.json" file or configure them directly as environment variables
+  - `languageWorkers__node__workerDirectory`: `<path to the root of this repository>`
+  - `languageWorkers__node__arguments`: `--inspect`
+    > NOTE: If you need to debug worker initialization, use `--inspect-brk` instead of `--inspect`. Just keep in mind you need to attach the debugger within 30 seconds or the host process will timeout.
+- Start the local function app (i.e. run `func start` or press <kbd>F5</kbd>)
+- Back in the worker repository, press <kbd>F5</kbd> and select the process for your running function app
+
```
Comment (alrod) [inline README.md:31]: Should we mention "npm test" to ensure all tests are passed?

Comment (alrod) [reply README.md:31]: Also maybe we need to add mocha profile so a contributor can set a breakpoint in a test

Comment (ejizba) [reply README.md:31]: Done. We already have a launch profile people can use for debugging unit tests here: https://github.com/Azure/azure-functions-nodejs-worker/blob/v3.x/.vscode/launch.json#L10

**Comment by ejizba** [comment]
ping @alrod @hossam-nasr

**Review by hossam-nasr — APPROVED** [review]

**Pull request merged**